src/net_processing.cpp

```diff
@@ -124,6 +124,9 @@ namespace {
     /** Number of peers from which we're downloading blocks. */
     int nPeersWithValidatedDownloads = 0;
 
+    /** Number of outbound peers with m_protect_from_disconnect. */
+    int g_outbound_peers_with_protect_from_disconnect = 0;
```

Comment:
Remove since this is easily calculated by counting nodes with `m_protect_from_disconnect`?

Reply:
By looping over all the CNodeState objects when this value is needed?  I don't think that makes sense.  Note that tracking the value here is analogous to what we do with `nPeersWithValidatedDownloads` and `nPreferredDownload`.